.github/workflows/test.yml

```diff
@@ -46,6 +46,16 @@ jobs:
       - name: Run shfmt
         run: shfmt -d bin/*
 
+      - name: Install action-validator with asdf
+        uses: asdf-vm/actions/install@v1
+        with:
+          tool_versions: |
+            action-validator 0.1.2
+
+      - name: Lint Actions
+        run: |
+          find .github/workflows -type f \( -iname \*.yaml -o -iname \*.yml \) \
+            | xargs -I {} action-validator --verbose {}
```

Comment:
Does it make sense to validate the actions configuration *within an action?* Is that ever going to run unless the configuration is already valid?

Reply:
one action could theoretically validate another

Reply:
As Mathew said, the "lint actions" job validates that all the *other* actions/workflows in a repo aren't going to cause issues.  This can be particularly handy when some workflows only run in post-merge contexts, like making releases (or periodically), because GitHub won't explode until some ill-defined time in the deep, dark future.